client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -22,8 +22,8 @@ export const PluginPanel = React.createClass( {
 				<Card className="wpcom-plugin-panel__panel-footer" href={ standardPluginsLink }>
 					{ this.translate( 'View all standard plugins' ) }
 				</Card>
-				<PremiumPluginsPanel />
-				<BusinessPluginsPanel />
+				<PremiumPluginsPanel siteSlug={ siteSlug } />
+				<BusinessPluginsPanel siteSlug={ siteSlug } />
```

Comment:
> the siteSlug is propagated to build the link GA plan link. It's needed specify the site in the url.
http://calypso.localhost:3000/plans/features/google-analytics/<site>

@retrofox how do you feel about the interpolator? the problem if we include this here is that in order to test or reuse the `<BusinessPluginsPanel />` we now need to know that it expects the site slug _and_ provide one _probably_ when it's not necessary.

I think it was pretty clear what we accomplish by passing it in, but I was just challenging us to find a way that doesn't needlessly couple these components.